packages/firebase-frameworks/package.json

```
@@ -43,7 +43,7 @@
     "dist"
   ],
   "engines": {
-    "node": "^16.0.0 || ^18.0.0 || ^20.0.0 || ^22.0.0"
+    "node": "^16.0.0 || ^18.0.0 || ^20.0.0 || ^22.0.0 || ^24.0.0"
```

I'd say we should [match firebase-tools](https://github.com/firebase/firebase-tools/blob/c65d86b27ead455837cdb7a6c7464d5cad579ead/package.json#L75-L77) and keep only `>=20.0.0 || >=22.0.0 || >=24.0.0`. @jamesdaniels WDYT?
```suggestion
    "node": ">=20.0.0 || >=22.0.0 || >=24.0.0"
```